ONBOARDING.md

```diff
@@ -0,0 +1,43 @@
+# Onboarding
+
+1. Create a new blank unity 3D project using the 2019.4.26.f1 version
+
+2. Install the latest versions of the following github packages, using the UnityPackageManager:
+
+- [ThunderKit](https://github.com/PassivePicasso/ThunderKit) version 8.0.3 or Greater
+- [RoR2ImportExtensions](https://github.com/risk-of-thunder/RoR2ImportExtensions) version 1.3.7 or Greater
+- [RoR2EditorKit](https://github.com/risk-of-thunder/RoR2EditorKit) version 4.0.4 or Greater
+
+
+> 2b. Sidenote, it is important that you use the versioned versions instead of the pre-release or indev versions. To select the correct version, You can click the big green “<> Code” button, and then copy the https url. You can append an “#” followed by your desired version to specify what version to download.
```

Comment:
Looks like the formatting here is a bit odd on the github markdown renderer, please fix

Reply:
Technically formatting isnt weird, its just me being weird using an uncommon markdown note style